content/docs/architecture.md

```diff
@@ -0,0 +1,47 @@
+# KUDO Architecture
+
+## The Purpose of KUDO
+
+KUDO is built to help Dev/Ops teams managed Day 2 Operations of Services on Kubernetes, including stateful services through the management of operators. Day 2 in this context refers to the need to support more than just installation. KUDO is built to support upgrades of services along with backup, recovery and the needs of observability. Building Kubernetes operators requires deep knowledge of the underlying service in addition to a deep knowledge of Kubernetes APIs. KUDO is built as an abstraction such that developers can build an operator without focusing on keeping up with the fast past of Kubernetes API changes. To this end KUDO supports the following:
+
+* Create new KUDO operator packages
+* Package operators in tarballs
+* Search, Install, Uninstall operators
+* Perform backup, restore and upgrade plans
+* Invoke other arbitrary packaged operator plans
+* Creating operator repositories with index files
+
+## Architecture Diagram
+
+![KUDO Architecture Diagram](/images/kudo-architecture.jpg?10x20)
+
+## Components
+
+**kudoctl** or `kubectl-kudo` is a [kubectl plugin](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/) command-line client. The client is used to aid developers in creating KUDO operators and is used by operators to manage operators in a Kubernetes cluster. The client is capable of:
+
+* Operator development
```

Comment:
what does it mean that it's capable of development? 🤔 maybe it's rather providing tools to package already developed operator?